sparse/tests/test_dot.py

```diff
@@ -74,7 +74,6 @@ def test_tensordot(a_shape, b_shape, axes, a_format, b_format):
     # tests for return_type=np.ndarray
     sa_sb = sparse.tensordot(sa, sb, axes, return_type=np.ndarray)
 
-    assert_eq(a_b, sa_sb)
```

Comment:
Why was this line removed?

Reply:
I thought it was redundant but it is not. My bad